test/ProgressOnderwijsUtilsTests/TransactedLocalConnection.cs

```diff
@@ -17,7 +17,12 @@ public TransactedLocalConnection()
             Context = new SqlCommandCreationContext(new SqlConnection(Environment.GetEnvironmentVariable("CONNECTION_STRING") ?? @"Server = (localdb)\MSSQLLocalDB; Integrated Security = true"), 60, SqlCommandTracer.CreateAlwaysOffTracer());
             try {
                 Context.Connection.Open();
-                ParameterizedSql.TableValuedTypeDefinitionScripts.ExecuteNonQuery(Context);
+                SQL($@"
+                    set transaction isolation level serializable;

```

Comment:
Actually, this makes more sense here anyhow, since the serializableness was an attempt to work around race-conditions in our test-set here (xunit runs em in parallel).  And the hack doesn't even work, so I'm fine with just leaving it out entirely.

Reply:
(If anybody else feels like seeing if we can allow parallel db tests, be my guest!